lms/static/sass/xmodule/_headings.scss

```diff
@@ -88,29 +88,33 @@ $headings-base-color:                       $gray-d2;
     letter-spacing: 1px;
 }
 
-.xblock .xblock {
+// let's make these classnames available to the entire LMS!
+
+.hd-1,
```

Comment:
It makes sense to me to introduce a new file for Pattern Library compatibility if you want to do that. I just don't want to have global styles defined beneath xmodule because nobody will expect to find them there.